Ginger/GingerCoreNETUnitTest/GingerRunnerTests/GingerRunnerTest.cs

```diff
@@ -20,46 +20,64 @@
 using Amdocs.Ginger.Common;
 using Amdocs.Ginger.CoreNET.Execution;
 using Amdocs.Ginger.CoreNET.Repository;
+using Amdocs.Ginger.CoreNET.Run.RunSetActions;
+using Amdocs.Ginger.CoreNET.RunLib.CLILib;
 using Amdocs.Ginger.Repository;
 using Ginger.Run;
+using Ginger.Run.RunSetActions;
+using Ginger.SolutionGeneral;
 using GingerCore;
 using GingerCore.Actions;
+using GingerCore.Environments;
 using GingerCore.Platforms;
 using GingerCore.Variables;
 using GingerCoreNET.SolutionRepositoryLib.RepositoryObjectsLib.PlatformsLib;
 using GingerCoreNETUnitTest.WorkSpaceLib;
 using GingerTestHelper;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
 
 namespace UnitTests.NonUITests.GingerRunnerTests
-{    
+{
     [TestClass]
     [Level1]
     public class GingerRunnerTest
-    {        
+    {
         static BusinessFlow mBF;
         static GingerRunner mGR;
         static SolutionRepository SR;
+        static RunsetExecutor GMR;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'GMR' and declare it as a local variable in the relevant methods.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4629760)